servicecontrol/upgrades/servicecontrol1to2.md

```diff
@@ -0,0 +1,57 @@
+---
```

Comment:
Should this file be deleted?

Reply:
Urgh, they should be combined. I'll do that. Not sure how I could have missed this